src/components/dashboard/SafeAppsDashboardSection/SafeAppsDashboardSection.tsx

```diff
@@ -0,0 +1,53 @@
+import { useRouter } from 'next/router'
+import Typography from '@mui/material/Typography'
+import Grid from '@mui/material/Grid'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+
+import { WidgetContainer } from '../styled'
+import { useSafeApps } from '@/hooks/safe-apps/useSafeApps'
+import { AppCard, AppCardContainer } from '@/components/safe-apps/AppCard'
+import { AppRoutes } from '@/config/routes'
+
+const SafeAppsDashboardSection = () => {
+  const { rankedSafeApps } = useSafeApps()
+
+  return (
+    <WidgetContainer>
+      <Typography component="h2" variant="subtitle1" fontWeight={700} mb={2}>
+        Safe Apps
+      </Typography>
+
+      <Grid container rowSpacing={2} columnSpacing={2}>
+        {rankedSafeApps.map((rankedSafeApp) => (
+          <Grid key={rankedSafeApp.id} item xs={12} sm={6} md={4} xl={4}>
+            <AppCard safeApp={rankedSafeApp} />
+          </Grid>
+        ))}
+
+        <Grid item xs={12} sm={6} md={4} xl={4}>
+          <ExploreSafeAppsCard />
+        </Grid>
+      </Grid>
+    </WidgetContainer>
+  )
+}
+
+export default SafeAppsDashboardSection
+
+const ExploreSafeAppsCard = () => {
+  const router = useRouter()
+  const safeAppsLink = `${AppRoutes.apps}?safe=${router.query.safe}`
+
+  return (
+    <AppCardContainer url={safeAppsLink}>
+      <Box display="flex" flexDirection="column" alignItems="center" justifyContent="center" height="100%" gap={1}>
+        <img src="/images/explore.svg" alt="Add custom app icon" />
+
+        <Button variant="contained" size="small">
+          Explore Safe Apps
+        </Button>
+      </Box>
```

Comment:
I'd suggest using `PagePlaceholder` for this and then adding the `illustration-xyz-xyz` `class` to `explore.svg` so it's dark mode compatibile.

Reply:
`no-transactions.svg` is a good example if you need a reference.

Reply:
done! thanks!

![Captura de pantalla 2022-09-22 a las 16 41 07](https://user-images.githubusercontent.com/26763673/191780675-fc40cc9c-4cb7-4f54-aa68-236507e633c2.png)